src/meson.build

```diff
@@ -576,6 +576,25 @@ libharfbuzz_subset_dep = declare_dependency(
   dependencies: [m_dep])
 meson.override_dependency('harfbuzz-subset', libharfbuzz_subset_dep)
 
+hb_cairo_sources = [
+  'hb-cairo.c',
+  'hb-cairo-utils.c'
+]
+
+cairo_dep = dependency('cairo')
+
+libharfbuzz_cairo = library('harfbuzz-cairo', hb_cairo_sources,
```

Comment:
Where does it install header files?

Reply:
It was just a quick draft, not a fully worked out branch. If we want this, I'll fix it up

Reply:
Yes we definitely want this, given how complicated writing a COLRv1 renderer is.